### PR TITLE
remove borken config for wet only

### PR DESCRIPTION
**Pull request by pindge**
- revert wofs wet only style, 
- align c2 style wet only

### CODE REVIEW

**Comment by pindge** [comment]
@GypsyBojangles 

This was the style before, apart from `sea` and `water` every other bit was marked as `mask: true`, on terria they were not showing, however them on the config caused ows to generate that bit for the legend, so I just removed the `mask: true` config

@SpacemanPaul , so i guess its either ows shouldn't generate legend if `mask: true` in `valuemap` or this `wet only` style is not configured correctly.
```
style_wofs_obs_wet_only = {
    "name": "wet",
    "title": "Wet Only",
    "abstract": "Wet Only",
    "value_map": {
        "water": [
            {
                "title": "Invalid",
                "abstract": "Slope or Cloud",
                "flags": {
                    "or": {
                        "terrain_or_low_angle": True,
                        "cloud_shadow": True,
                        "cloud": True,
                        "high_slope": True,
                        "noncontiguous": True,
                    }
                },
                "color": "#707070",
                "mask": True,
            },
            {
                # Possible Sea Glint, also mark as invalid
                "title": "",
                "abstract": "",
                "flags": {"dry": True, "sea": True},
                "color": "#707070",
                "mask": True,
            },
            {
                "title": "Dry",
                "abstract": "Dry",
                "flags": {
                    "dry": True,
                    "sea": False,
                },
                "color": "#D99694",
                "mask": True,
            },
            {
                "title": "Wet",
                "abstract": "Wet or Sea",
                "flags": {"or": {"wet": True, "sea": True}},
                "color": "#4F81BD",
            },
        ]
    },
}
```

**Comment by SpacemanPaul** [comment]
I believe it should not generate legend if mask is True.  If it doesn't (which would not surprise me), open it as an ows issue.

File: dev/services/wms/ows_refactored/wofs/ows_wofs_cfg.py [hunk 1]
```diff
@@ -130,23 +130,23 @@
                 "title": "",
                 "abstract": "",
                 "flags": {"noncontiguous": True,},
-                "mask": True,    
+                "mask": True,
```
Comment (SpacemanPaul) [inline dev/services/wms/ows_refactored/wofs/ows_wofs_cfg.py:133]: May as well fix it here too.

**Review by SpacemanPaul — CHANGES_REQUESTED** [review]

**Review by SpacemanPaul — APPROVED** [review]
Looks good.

**Pull request merged**